flygym/mujoco/examples/common.py

```diff
@@ -31,6 +31,9 @@ class PreprogrammedSteps:
     path : str or Path, optional
         Path to the preprogrammed steps data. If None, the default
         preprogrammed steps data will be loaded.
+    neutral_pos_indexes : List[float]
```

Comment:
I would call it `neutral_pose_phases` because (1) pos can be misunderstood as either cartesian position or a set of joint angles, and (2) if it's normalized to [0, 2pi) it's not really an index but a phase.